ansible_collections/arista/avd/docs/how-to/first-project.md

```diff
@@ -253,15 +250,15 @@ In each variable file related to a type of devices, we have to instruct AVD what
 
 ```yaml
 ---
-type: spine     # Must be either spine|l3leaf|l2leaf
+type: spine     # Must be either spine | l3leaf | l2leaf
 ```
 
-### Configure VNI/VLAN across the Fabric.
+### Configure VNI/VLAN across the Fabric
 
-AVD supports mechanism to create VLANs and VNIs and enable traffic forwarding in your overlay. In current version (`v1.0.2`), only following design listed below are supported:
+AVD supports mechanism to create VLANs and VNIs and enable traffic forwarding in your overlay. In current version (`v3.1.0`), only following design listed below are supported:
```

Comment:
We are already in newer versions. I suggest we delete the reference to a specific version in documentation.

Reply:
ok